src/games/strategy/triplea/ui/MapData.java

```diff
@@ -700,6 +701,50 @@ private Rectangle getBoundingRectWithTranslate(final List<Polygon> polys, final
   }
 
   /**
+   * Find the best rectangle inside the territory to place the name in. Finds the rectangle
+   * that can fit the name, that is the closest to the vertical center, and has a large width at
+   * that location. If there isn't any rectangles that can fit the name then default back to the
+   * bounding rectangle.
+   */
+  public Rectangle getBestTerritoryNameRect(final Territory t, final FontMetrics fm) {
+    final Rectangle territoryBounds = getBoundingRect(t);
+    Rectangle maxRectangle = territoryBounds;
```

Comment:
I'd like to leave territoryBounds first since that is what the function is returning. I've renamed the return variable to 'result'.